clients/src/main/java/org/apache/kafka/common/requests/AbstractResponse.java

```diff
@@ -125,4 +125,7 @@ public static AbstractResponse getResponse(ApiKeys apiKey, Struct struct) {
         }
     }
 
+    public String toString(short version) {
```

Comment:
I was thinking that we can add a default `toString` for the latest version as long as we make it clear in the output. It's still better than `Object.toString`. What do you think?